### ISSUE TITLE
generated .d.ts change import 'moduleA' to import '../../../../moduleA'

### ISSUE THREAD

**Issue by whu-iss-jack** [Microsoft/TypeScript#25543]
index.ts
```ts
import { combineReducers } from 'redux';
import count from './count';

export default combineReducers({ count });
```
index.d.ts
```ts
declare const _default: import("../../../../../../../Users/bgfist/Documents/Shimo/racoon/node_modules/redux").Reducer<{
    count: {
        numOne: number;
        label: string;
    };
}, import("../../../../../../../Users/bgfist/Documents/Shimo/racoon/node_modules/redux").AnyAction>;
export default _default;
```
actually, I don't add any types to my export because typescript can infer it, and the generated .d.ts importd the missing types for me, but why it's relative?

**Comment by whu-iss-jack** [comment]
@kitsonk ah, I'm sorry, I rarely commit issues. So this fix is updated in which typescript version?

**Comment by kitsonk** [comment]
It should be fixed in `typescript@next` and 3.0 when released.

**Comment by whu-iss-jack** [comment]
@kitsonk ok, thanks
